lib/lhm/migrator.rb

```diff
@@ -196,6 +196,7 @@ def destination_read
     end
 
     def index_ddl(cols, unique = nil, index_name = nil)
+      assert_valid_idx_name(index_name)
```

Comment:
@camilo since this is already merged upstream I'd rather leave it as in. `assert_*` is a thing in Rails to validate arguments, see [`Hash#assert_valid_keys`](http://devdocs.io/rails/hash#method-i-assert_valid_keys).